devicetypes/stelpro/stelpro-ki-thermostat.src/stelpro-ki-thermostat.groovy

```diff
@@ -50,11 +50,12 @@ metadata {
 
 	preferences {
 		section {
-			input("heatdetails", "enum", title: "Do you want a detailed operating state notification?", options: ["No", "Yes"], defaultValue: "No", required: true, displayDuringSetup: true)
+			input("heatdetails", "enum", title: "Do you want to see detailed operating state events in the activity history? There may be many.", options: ["No", "Yes"], defaultValue: "No", required: false, displayDuringSetup: true)
 		}
 		section {
-			input title: "Outdoor Temperature", description: "To get the current outdoor temperature to display on your thermostat enter your zip code or postal code below and make sure that your SmartThings location has a Geolocation configured (typically used for geofencing).", displayDuringSetup: false, type: "paragraph", element: "paragraph"
-			input("zipcode", "text", title: "ZipCode (Outdoor Temperature)", description: "[Do not use space](Blank = No Forecast)")
+			input(title: "Outdoor Temperature", description: 	"To get the current outdoor temperature to display on your thermostat enter your zip code or postal code below and make sure that your SmartThings location has a Geolocation configured (typically used for geofencing)." +
```

Comment:
Any reason for the odd formatting? I'd take out the tab after `description:` and just combine the string constant instead of using `+` -- but don't forget to add a space in the string before the "Do not" part.

Reply:
Nah, generally I don't like too long lines, so that's the reason for this